src/dice/DamageRoll.ts

```diff
@@ -82,9 +100,10 @@ class DamageRoll extends foundry.dice.Roll<DamageRoll.Data> {
 					// Create Primary Term
 					primaryTerm = new PrimaryDie({
 						number: 1 + Math.abs(rollMode),
-						faces,
+						faces: faces ?? 6,
+						modifiers: [],
 						options: { flavor: 'Primary Die' },
-					});
+					} as object as foundry.dice.terms.Die.TermData);
```

Comment:
Why isn't this defined as the type returned by PrimaryDie instead of a cast type?